Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+
+function CardContent(props) {
```

Comment:
This is fine but you don't really need to declare props in your functional component if you are not passing any.
Like any JS function if you don't need to use outside arguments in your body, you don't need to declare any, it makes your code easier to read.